### PR TITLE
48942: the comment for the versions is added to the activity of the initial document

### PR DESCRIPTION
**Pull request by wassimmhamdi**
Problem: The comment of the version of the documentY is added in the activity of the documentX
Fix: Version comment of documentY are added to the correct activity of documentY

### CODE REVIEW

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java [hunk 1]
```diff
@@ -151,23 +152,22 @@ public void execute(Event<UIDocumentAutoVersionForm> event) throws Exception {
...
+            int lastDotIndex = destPath.lastIndexOf('.');
+            while (existNode != null) {
+              newDestPath = destPath.substring(0, lastDotIndex) + "-" + (fileIndex + 1 ) + destPath.substring(lastDotIndex);
+              existNode = uiExplorer.getNodeByPath(newDestPath, srcSession);
+              fileIndex ++;
             }
+            destPath = newDestPath;
+            copyTitle = destPath.substring(destPath.lastIndexOf("/") + 1, lastDotIndex) + "(" + (fileIndex + 1 ) + ")" + destPath.substring(lastDotIndex);
+            }
+
```
Comment (rdenarie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java]: The formatting is not ok

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java (hunk 1 above)
Comment (wassimmhamdi) [reply core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java]: I formatted it

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java [hunk 2]
```diff
@@ -166,8 +166,8 @@ public void execute(Event<UIDocumentAutoVersionForm> event) throws Exception {
               }
               destPath = newDestPath;
               copyTitle = destPath.substring(destPath.lastIndexOf("/") + 1, lastDotIndex) + "(" + (fileIndex + 1 ) + ")" + destPath.substring(lastDotIndex);
-
             }
+
```
Comment (rdenarie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java]: Please Remove theses 2 modifications

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java [hunk 3]
```diff
@@ -151,7 +152,6 @@ public void execute(Event<UIDocumentAutoVersionForm> event) throws Exception {
             PasteManageComponent.pasteByCut(autoVersionComponent.getCurrentClipboard(), uiExplorer, destSession, autoVersionComponent.getCurrentClipboard().getWorkspace(),
                     sourceNode.getPath(), destPath, WCMCoreUtils.getService(ActionServiceContainer.class), false,false, false);
           }else {
-            // nt:folder Does not support SNS
```
Comment (rdenarie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java]: Keep also this comment

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java (hunk 2 above)
Comment (wassimmhamdi) [reply core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java]: I removed it

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java (hunk 3 above)
Comment (wassimmhamdi) [reply core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java]: I added the comment **// nt:folder Does not support SNS** to the code

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java (hunk 2 above)
Comment (rdenarie) [reply core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java]: you still have a line remove. Please check the diff and readd it

Comment (wassimmhamdi) [reply core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java]: I check it and I correct it

**Review by rdenarie — APPROVED** [review]

**Pull request merged**